jwebunit-htmlunit-plugin/src/main/java/net/sourceforge/jwebunit/htmlunit/HtmlUnitTestingEngineImpl.java

```diff
@@ -1502,25 +1495,27 @@ public HtmlElement getButtonWithText(String buttonValueText) {
       throw new NullPointerException("Cannot search for button with null text");
     }
 
-    List<? extends HtmlElement> l = ((HtmlPage) win.getEnclosedPage()).getDocumentElement()
-      .getHtmlElementsByTagNames(

```

Comment:
This method was removed from htmlunit and as suggested a simple iteration over a list should be made